src/cplscheme/impl/MinIterationConvergenceMeasure.hpp

```diff
@@ -35,10 +35,10 @@ class MinIterationConvergenceMeasure : public ConvergenceMeasure {
     return _isConvergence;
   }
 
-  virtual std::string printState()
+  virtual std::string printState(const std::string &dataName)
   {
     std::ostringstream os;
-    os << "min iteration convergence measure: ";
+    os << "min iteration convergence measure of data \"" << dataName << "\": ";
```

Comment:
I guess it's debatable whether we want it here. Of course, it's already weird / wrong that a min-iteration convergence measure requires a data field :see_no_evil: . But it does not get better if we output it.

Reply:
I removed it